catalyst/loggers/mlflow.py

```diff
@@ -0,0 +1,189 @@
+from typing import Any, Dict, Optional
+
+import mlflow
+import numpy as np
+
+from catalyst.core.logger import ILogger
+from catalyst.loggers.functional import (
+    EXCLUDE_PARAMS,
+    EXPERIMENT_PARAMS,
+    mlflow_log_dict,
+    STAGE_PARAMS,
+)
+
+
+class MlflowLogger(ILogger):
```

Comment:
could you please add it to the docs? https://github.com/catalyst-team/catalyst/blob/dev/docs/api/loggers.rst

Reply:
and could you please write some tests for it? for example, you could add it to our `test_finetuning` pipeline https://github.com/catalyst-team/catalyst/blob/dev/catalyst/tests/test_finetune2.py#L29

Reply:
done